lib/ansible/modules/network/cloudengine/ce_evpn_bgp_rr.py

```diff
@@ -0,0 +1,534 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: ce_evpn_bgp_rr
+version_added: "2.3"
+short_description: Manages RR for the VXLAN Network.
+extends_documentation_fragment: cloudengine
+description:
+    - Configure an RR in BGP-EVPN address family view.
+author: Zhijin Zhou (@CloudEngine-Ansible)
+notes:
+    - Ensure that BGP view is existed.
+    - The peer, peer_type, and reflect_client arguments must all exist or not exist.
+options:
+    as_number:
+        description:
+            - Specifies the number of the AS, in integer format.
+              The value is an integer that ranges from 1 to 4294967295.
+        required: true
+        default: null
+    bgp_instance:
+        description:
+            - Specifies the name of a BGP instance.
+              The value of instance-name can be an integer 1 or a string of 1 to 31.
+        required: false
+        default: null
+    bgp_evpn_enable:
+        description:
+            - Enable or disable the BGP-EVPN address family.
+        required: false
+        choices: ['true','false']
+        default: true
+    peer_type:
+        description:
+            - Specify the peer type.
+        required: false
+        choices: ['group_name','ipv4_address']
+        default: null
+    peer:
+        description:
+            - Specifies the IPv4 address or the group name of a peer.
+        required: false
+        default: null
+    reflect_client:
+        description:
+            - Configure the local device as the route reflector and the peer or peer group as the client of the route reflector
+        required: false
+        choices: ['true','false']
+        default: null
+    policy_vpn_target:
+        description:
+            - Enable or disable the VPN-Target filtering.
+        required: false
+        choices: ['true','false']
+        default: null
+'''
+
+EXAMPLES = '''
+# Configure BGP-EVPN address family view and ensure that BGP view has existed.
```

Comment:
All examples should specify a `- name:` rather than a `#`
Please see http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#examples-block for more details.